qemu/tests/cfg/device_assignment.cfg

```diff
@@ -1,13 +1,26 @@
 - device_assignment:
     only pf_assignable
     virt_test_type = qemu
+    pre_command = "modprobe vfio-pci"
```

Comment:
In test_setup.py, function sr_iov_setup() will check this, no need to add it here.
If sr_iov_setup() have bug  for this operation, bug fix is enough, i think.

And If can not reuse sr_iov_setup at last, move this to guest_hw.cfg for "pf_assignable" and "vf_assignable" is better.